README.md

```diff
@@ -100,7 +139,8 @@ We performed different experiments using random and time splits on the reaction
 
 ## Citation
 
-If you found this repo useful, please cite our [manuscript](https://chemrxiv.org/articles/preprint/Prediction_of_Chemical_Reaction_Yields_using_Deep_Learning/12758474).
+If you found this repo useful, please cite our the following preprints:
```

Comment:
"cite the following preprints" (delete "our")

Reply:
Thanks!